test/functional/autocmd/completedone_spec.lua

```diff
@@ -0,0 +1,41 @@
+local t = require('test.testutil')
+local n = require('test.functional.testnvim')()
+
+local clear = n.clear
+local command = n.command
+local call = n.call
+local feed = n.feed
+local eval = n.eval
+local eq = t.eq
+
+describe('CompleteDone', function()
+  before_each(clear)
+
+  describe('sets correct reason in v:event', function()
```

Comment:
"correct", "works", "should", are always redundant in test names.
```suggestion
  describe('sets v:event.reason', function()
```